### PR TITLE
fix(ci): free up disk space

### PR DESCRIPTION
**Pull request by crenshaw-dev**
Need some of the large packages stuff. https://cloud-native.slack.com/archives/C020XM04CUW/p1695759346952319?thread_ts=1695737319.502399&cid=C020XM04CUW

### CODE REVIEW

**Review by ishitasequeira — APPROVED** [review]

File: .github/workflows/image-reuse.yaml [hunk 1]
```diff
@@ -137,6 +137,8 @@ jobs:
 
       - name: Free Disk Space (Ubuntu)
         uses: jlumbroso/free-disk-space@4d9e71b726748f254fe64fa44d273194bd18ec91
+        with:
+          large-packages: false
```
Comment (34fathombelow) [inline .github/workflows/image-reuse.yaml:141]:
```suggestion
          large-packages: false
          docker-images: false
          swap-storage: false
```

**Review by 34fathombelow — COMMENTED** [review]
I think we should also add these to be safe.

File: .github/workflows/image-reuse.yaml [hunk 2]
```diff
@@ -137,6 +137,10 @@ jobs:
...
+          docker-images: false
+          swap-storage: false
```
Comment (34fathombelow) [inline .github/workflows/image-reuse.yaml:143]:
```suggestion
          swap-storage: false
          tool-cache: false
```

Comment (crenshaw-dev) [inline .github/workflows/image-reuse.yaml:143]:
```suggestion
          swap-storage: false
          tool-cache: false
```

**Review by 34fathombelow — COMMENTED** [review]
also this one and we should be good togo

**Pull request merged**